addons/mail/static/src/core_ui/link_preview/link_preview.scss

```diff
@@ -20,6 +20,11 @@
     max-width: $o-mail-LinkPreview-width;
 }
 
+.o-mail-ChatWindow-content .o-mail-LinkPreviewImage img {
+    max-height: $o-mail-LinkPreview-height;
+    max-width: 100%;
```

Comment:
Shouldn't the basic LinkPreviewImage rule have Min(value, 100%) instead to cover both cases?